pkg/util/pki/fetchdata.go

```diff
@@ -0,0 +1,33 @@
+package pki
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"encoding/json"
+	"io"
+	"net/http"
+)
+
+// https://aka.ms/getissuers
+// The v3 endpoint can be used to get ca certs
+// For example https://issuer.pki.azure.com/dsms/issuercertificates?getissuersv3&caName=ame
+// returns the ame certs
+func FetchDataFromGetIssuerPki(url string) (*RootCAs, error) {
+	response, err := http.Get(url)
+	if err != nil {
+		return nil, err
+	}
+
+	defer response.Body.Close()
+
+	// Read in certs from endpoint
```

Comment:
Do we need this comment?

Reply:
It was there, but we don't need it. Removed. Thanks.